Jellyfin.Server/Migrations/Routines/RemoveBuggedExtras.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using System.IO;
+
+using MediaBrowser.Controller;
+using Microsoft.Extensions.Logging;
+using SQLitePCL.pretty;
+
+namespace Jellyfin.Server.Migrations.Routines
+{
+    /// <summary>
+    /// Remove duplicate entries which were caused by a bug where a file was considered to be an "Extra" to itself.
+    /// </summary>
+    internal class RemoveBuggedExtras : IMigrationRoutine
```

Comment:
I think something that explains what is 'bugged' would be a bit better

```suggestion
    internal class RemoveDuplicateExtras : IMigrationRoutine
```

Reply:
Sorry, should have clarified that this suggestion can't be applied directly. You'll need to rename the file and constructor as well and probably also want to update the `Name` property